src/app/data-analytics-dashboard/oncology/oncology-reports/oncology-report-pdf-view/oncology-report-pdf-view.component.spec.ts

```diff
@@ -0,0 +1,357 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { NO_ERRORS_SCHEMA, DebugElement } from '@angular/core';
+import { of } from 'rxjs';
+import { By } from '@angular/platform-browser';
```

Comment:
Move this line to the top of the page. Just after angular core. Then have a space line after it